src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -19,18 +19,18 @@
     <!-- the following package(s) are from https://github.com/dotnet/fxdac -->
     <PackageReference Include="System.Data.SqlClient" Version="4.8.5" />
     <!-- the following package(s) are from https://github.com/dotnet/corefx -->
-    <PackageReference Include="System.IO.Packaging" Version="8.0.0-preview.4.23259.5" />
-    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="8.0.0-preview.4.23259.5" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="8.0.0-preview.4.23259.5" />
+    <PackageReference Include="System.IO.Packaging" Version="8.0.0-preview.7.23375.6" />
+    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="8.0.0-preview.7.23375.6" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="8.0.0-preview.7.23375.6" />
     <!-- the following package(s) are from https://github.com/dotnet/wcf -->
-    <PackageReference Include="System.ServiceModel.Duplex" Version="4.10.2" />
-    <PackageReference Include="System.ServiceModel.Http" Version="4.10.2" />
-    <PackageReference Include="System.ServiceModel.NetTcp" Version="4.10.2" />
-    <PackageReference Include="System.ServiceModel.Primitives" Version="4.10.2" />
-    <PackageReference Include="System.ServiceModel.Security" Version="4.10.2" />
+    <PackageReference Include="System.ServiceModel.Duplex" Version="6.0.0" />
+    <PackageReference Include="System.ServiceModel.Http" Version="6.0.0" />
+    <PackageReference Include="System.ServiceModel.NetTcp" Version="6.0.0" />
+    <PackageReference Include="System.ServiceModel.Primitives" Version="6.0.0" />
+    <PackageReference Include="System.ServiceModel.Security" Version="6.0.0" />
```

Comment:
The `System.ServiceModel.xxx` packages were pinned to the 4.10.2 because the new versions cause breaking changes to PowerShell. Did you verified its been fixed in 6.0.0?

https://github.com/PowerShell/PowerShell/issues/19238

Reply:
Will revert the change.